dist/testbed-support/Makefile.iotlab

```diff
@@ -55,7 +55,10 @@ iotlab-reset: $(IOTLAB_AUTH) iotlab-check-exp
 	$(AD)node-cli --reset -i $(IOTLAB_EXP_ID) $(NODES_PARAM_BASE) $(EXCLUDE_PARAM)
 
 iotlab-debug-server: $(IOTLAB_AUTH) iotlab-check-exp
-	$(AD)node-cli --debug-start -i $(IOTLAB_EXP_ID)
+	$(eval DEBUG_TYPE := $(shell echo $(IOTLAB_TYPE) | cut -d: -f1))
+	$(eval DEBUG_NODE := $(shell echo $(IOTLAB_DEBUG_NODE) | sed 's/$(DEBUG_TYPE)-\([0-9]*\)/\1/'))
```

Comment:
Not really .. I am fine with the current solution. I was just wondering, because `-l $(IOTLAB_SITE),$(DEBUG_TYPE),$(DEBUG_NODE)` looks awfully similar to how `NODES_PARAM_BASE` is defined when `IOTLAB_PHY_NODES` is used.

Nevertheless, don't mind me.